docs/tutorials/basics.md

```diff
@@ -262,7 +262,7 @@ Route creation steps:
 
 ## Building skipper
 
-We use Go modules to build skipper, therefore you need [Go](https://golang.org/dl) version `>= 1.11`.
+Skipper uses go modules and to build it you'll need [Go](https://golang.org/dl) version `>= 1.19`.
```

Comment:
Lets maybe remove this sentence so we don't need to update it in the future? WDYT?

Reply:
Or just say it requires `go`

Reply:
> Lets maybe remove this sentence so we don't need to update it in the future? WDYT?

Makes sense.